lib/galaxy/tool_util/xsd/galaxy.xsd

```diff
@@ -2980,10 +2984,10 @@ between paired inputs of single stranded inputs.
     <xs:complexContent>
       <xs:extension base="InputType">
         <xs:sequence>
-          <xs:group ref="InputElement" minOccurs="0" maxOccurs="1" />
+          <xs:element name="param" type="Param" minOccurs="1" maxOccurs="1" />
```

Comment:
Seems that we have to revert to `minOccurs="1"` because the upload tool uses a conditional without param ... or?

Reply:
Probably ? It does only work when value_ref and value_from are used, which noone should be using, so if you want to exclude the upload tool from linting that'd be fine with me too.